docs/ru/operations/clickhouse-keeper.md

```diff
@@ -1,4 +1,4 @@
----
+[---
```

Comment:
That's strange.

Reply:
@alexey-milovidov What do you mean? Russian article very old (

Reply:
You replaced the whole article with the link to your personal profile.